java/dataset/pom.xml

```diff
@@ -109,6 +109,38 @@
             <artifactId>jackson-databind</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.arrow.orc</groupId>
+            <artifactId>arrow-orc</artifactId>
```

Comment:
Is this dependency actually used? I see the ORC library but not the Arrow ORC adapter

Reply:
@lidavidm sure! If comment this dependency: java.lang.NoClassDefFoundError: org/apache/hadoop/hdfs/client/HdfsDataOutputStream$SyncFlag
![image](https://user-images.githubusercontent.com/10332206/188971638-b8c4b8eb-9074-445e-b57d-4b5a71c0655d.png)

Reply:
It _seems_ like that's just a transitive dependency that needs to get pulled in…but since this is a test-only dependency it's ok then.